x-pack/plugins/ingest_manager/server/services/epm/elasticsearch/ilm/ilm.ts

```diff
@@ -13,26 +13,3 @@ export function getIndexWithWithAlias(aliasName: string) {
     },
```

Comment:
@ruflin Not sure what this "getIndexWithWithAlias" function is doing. Can it be removed?
ee00808#diff-2f7a49f58cea808667dd6faca14721ebL7

Reply:
This was intended to create the content for creating an index with an Alias (required for rollover / ILM). `WithWith` is  typo. As we will not need to setup any alias in the future I suggest to remove this or if we realise we need it, reimplement it.